pkg/election/leadership.go

```diff
@@ -62,6 +63,9 @@ type Leadership struct {
 	keepAliveCtx            context.Context
 	keepAliveCancelFunc     context.CancelFunc
 	keepAliveCancelFuncLock syncutil.Mutex
+	// CampaignTimes is used to record the campaign times of the leader in 5min.
+	// To avoid the leader campaign too frequently.
+	CampaignTimes []cache.TTLString
```

Comment:
TTLString is container, no needs to use array.BTW, you can only use array not ttlcache.

Reply:
good catch. fixed